source/includes/fact-match-dump-restore-versions.rst

```diff
@@ -1,9 +1,26 @@
 When using :binary:`~bin.mongorestore` to load data files created by
-:binary:`~bin.mongodump`, be sure that you are restoring to the *same
-major version* of the MongoDB Server that the files were created from.
-For example, if your dump was created from a MongoDB Server running
-version ``4.4.x``, be sure that the MongoDB Server you are restoring to
-is also running version ``4.4.x``.
+:binary:`~bin.mongodump`, be sure that the deployment you are restoring 
+to has either:
+
+- The same major version.
+- The same feature compatibility version. 
```

Comment:
[suggestion]

I think we can word this in a slightly more direct way. What about:

> When using mongorestore to load data files created by mongodump, the MongoDB versions of your source and destination deployments must be either:
> 
> - The same major version
> - The same feature compatibility version

This is slightly wordier, but I think helps reduce ambiguity.